ts/screens/profile/NewProfileScreen.tsx

```diff
@@ -0,0 +1,133 @@
+import React from "react";
+import { IconProps } from "react-native-vector-icons/Icon";
+import { List as NBList } from "native-base";
+import { useDispatch } from "react-redux";
+import * as pot from "@pagopa/ts-commons/lib/pot";
+
+import { emptyContextualHelp } from "../../utils/emptyContextualHelp";
+import ScreenContent from "../../components/screens/ScreenContent";
+import I18n from "../../i18n";
+import TopScreenComponent from "../../components/screens/TopScreenComponent";
+import { HEADER_ICON_HEIGHT } from "../../utils/constants";
+import { useOnFirstRender } from "../../utils/hooks/useOnFirstRender";
+import ListItemComponent from "../../components/screens/ListItemComponent";
+
+import NameSurnameIcon from "../../../img/assistance/nameSurname.svg";
+import FiscalCodeIcon from "../../../img/assistance/card.svg";
+import EmailIcon from "../../../img/assistance/email.svg";
+import { loadNewProfile } from "../../store/actions/newProfile";
+import { newProfileSelector } from "../../store/reducers/newProfile";
+import { LoadingErrorComponent } from "../../features/bonus/bonusVacanze/components/loadingErrorScreen/LoadingErrorComponent";
+import GenericErrorComponent from "../../components/screens/GenericErrorComponent";
+import { useIOSelector } from "../../store/hooks";
+import { InitializedProfile } from "../../../definitions/backend/InitializedProfile";
+import { getPrintableValueFromPot } from "../../utils/pot";
+
+const newProfileScreenIconProps: IconProps = {
+  name: "io-profilo",
+  size: HEADER_ICON_HEIGHT
+};
+
+const screenTitle = I18n.t("profile.main.title");
+
+const NewProfileScreen = () => {
+  const dispatch = useDispatch();
+
+  const newProfilePot = useIOSelector(newProfileSelector);
+
+  useOnFirstRender(() => {
+    loadProfile();
+  });
+
+  /**
+   * Function that start fetch the profile informations
+   */
+  const loadProfile = () => {
+    dispatch(loadNewProfile.request());
+  };
+
+  /**
+   * Wrapper function of `getPrintableValueFromPot` to get the value from the pot with a fallback value
+   */
+  const getValueFromNewProfilePot = (value: keyof InitializedProfile) =>
+    getPrintableValueFromPot(
+      newProfilePot,
+      value,
+      I18n.t("global.remoteStates.notAvailable")
+    );
+
+  /**
+   * Wrapper component to show the loading view
+   */
+  const LoadingView = () => (
+    <LoadingErrorComponent
+      isLoading
+      loadingCaption={I18n.t("global.remoteStates.loading")}
+      onRetry={() => null}
+    />
+  );
+
+  /**
+   * Wrapper component to show the error view with the retry button to reload the profile
+   */
+  const ErrorView = () => (
+    <GenericErrorComponent onRetry={() => loadProfile()} />
+  );
+
+  const ProfileContentView = () => (
+    <ScreenContent iconFont={newProfileScreenIconProps} title={screenTitle}>
+      <NBList withContentLateralPadding>
```

Comment:
Native base is planned to be dismissed, why not using the common just a simple `ScrollView`?

Reply:
Yep! I saw just right now the Damiano's topic about the dismission of Native Base, so I've changed it with the `ScrollView` as suggested by you with a common style that adds horizontal padding to its content  ([996d1b4](https://github.com/Hantex9/io-app/pull/1/commits/996d1b4f0264ae57385fc7be4a30892fd2da19ce))